includes/controls/groups/background.php

```diff
@@ -258,6 +258,19 @@ public function init_fields() {
 			],
 		];
 
+		$fields['dimension'] = [
+			'label' => esc_html_x( 'Dimension', 'Background Control', 'elementor' ),
```

Comment:
```suggestion
			'label' => esc_html_x( 'Dimensions', 'Background Control', 'elementor' ),
```

Reply:
done

Reply:
Where is the change?

Reply:
Not relevant anymore, the logic has changed.